static/app/components/searchQueryBuilder/index.stories.tsx

```diff
@@ -715,6 +715,83 @@ export default Storybook.story('SearchQueryBuilder', story => {
     );
   });
 
+  story('Match key suggestions', () => {
+    return (
+      <Fragment>
+        <p>
+          If you would like to show suggestions for keys when the user types a value, you
+          can do so by passing the <code>matchKeySuggestions</code> prop, which requires a
+          key and a value regex pattern.
```

Comment:
Not blocking, but it just occurred to me that it might be better to accept a predicate function `(value: str) => bool` instead of a regex since that would allow for use of a parser or something other than regex if needed. Though it's not a big deal since regex probably supports all our use cases for now